src/components/Dashboard.vue

```diff
@@ -192,7 +194,9 @@ export default {
       }
 
       // Distribution of course (e.g. MQR-AS)
-      const distributions = course.distributions || course.catalogDistr.split(',');
+      // alternateDistributions option in case catalogDistr for the course is null
+      const alternateDistributions = (course.catalogDistr != null) ? /\(([^)]+)\)/.exec(course.catalogDistr)[1].split(', ') : [""];
```

Comment:
course.catalogDistr != null does not account for when course.catalogDistr is undefined. Consider making it either just (course.catalogDistr) ? ... or (course.catalogDistr === undefined || course.catalogDistr === nulll)

Reply:
The first suggestion @hanywang2 made will work as both null and undefined evaluate to false. Did not know this before, thanks for the suggestion!

Reply:
Thanks for the suggestions! I had to end up doing the second way
(course.catalogDistr || course.catalogDistr === '') because there were some cases when course.catalogDistr could have been ''. I did the same changes for catalogWhenOffered.